modules/openapi-generator/src/main/resources/dart2/model.mustache

```diff
@@ -1,12 +1,8 @@
-part of {{pubName}}.api;
-
 {{#models}}
-{{#model}}
-{{#isEnum}}
-{{>enum}}
-{{/isEnum}}
-{{^isEnum}}
-{{>class}}
-{{/isEnum}}
-{{/model}}
-{{/models}}
+    {{#model}}
+        {{#imports}}
+            import '{{.}}';
+        {{/imports}}
+        {{#isEnum}}{{>enum}}{{/isEnum}}{{^isEnum}}{{>class}}{{/isEnum}}
+    {{/model}}
```

Comment:
Should we remove these leading spaces (line 2 to 7)?

Reply:
Yeah good call, I'll make the change now.  My troubled local branch seems to have magically come good btw.